sdk/ml/azure-ai-ml/tests/pipeline_job/e2etests/test_pipeline_job.py

```diff
@@ -1900,6 +1900,22 @@ def check_name_version_and_register_succeed(output, asset_name):
         assert pipeline_job.jobs["compare"].outputs.best_model.name == "best_model"
         assert pipeline_job.jobs["compare"].outputs.best_model.version == random_version
 
+    @pytest.mark.skipif(condition=not is_live(), reason="Task 2177353: component version changes across tests.")
+    @pytest.mark.parametrize(
+        "yaml_path",
+        [
+            "./tests/test_configs/pipeline_jobs/serverless_compute/all_types/command/pipeline.yml",
```

Comment:
Maybe re-org like this:
./tests/test_configs/pipeline_jobs/all_types/command/pipeline_serverless_compute.yml

I think we might add more such cross component type pipeline feature

Reply:
Sure, will do this in next PR.